### ISSUE TITLE
FR: Persistent Undos

### ISSUE THREAD

**Issue by chrisgrieser** [antoniotejada/obsidian-edit-history#3]
One pretty awesome feature that I know from a different app (vim) is the concept of so-called [persistent undos](https://neovim.io/doc/user/undo.html#undo-persistence), which could be an interesting feature for this plugin.

The idea is you make your undo history persist across sessions by storing them in edit history files, not too dissimilar from this plugin. Based on this plugin's settings, the granularity of undos would get lost, but it's still a big plus compared to not having anything.

Basically, what this plugin would have to do is somehow "hook into" Obsidian's `ctrl-z` (or `cmd-z`) when Obsidian reaches the end of its own undo history.

**Comment by antoniotejada** [comment]
Haha great minds think alike. 

There's a todo about that in the code:
```
// XXX Feed the editor undo stack with the contents of the history file? (could
//     be done without private apis by inserting the text in edit history order at file
//     load, will probably need a flag to prevent from storing double history)
``
https://github.com/antoniotejada/obsidian-edit-history/blob/6d55e99c4ff098d8a9ec2e5dbe5d56420278a7f4/main.ts#L71C1-L73C79

Leaving open to consider implementing.
